tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_devcenter_26189.go

```diff
@@ -0,0 +1,77 @@
+package dataworkarounds
+
+import (
+	"fmt"
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
+)
+
+// This workaround fixes an issue in DevCenter where the field `DevCenterUri` is not
+// marked as ReadOnly, meaning that it's surfaced as an Optional field rather than
+// being Computed.
```

Comment:
You're also setting the DevCenterId field to Required here as well, could you add that in the description here?

Reply:
Good shout, will fix that now.